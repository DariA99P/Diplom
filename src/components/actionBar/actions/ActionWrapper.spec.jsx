import React from 'react';
import { shallow } from 'enzyme';

import ActionWrapper from '.';

describe('<ActionWrapper />', () => {
  let tree;
  beforeEach(() => {
    tree = shallow(<ActionWrapper />);
  });

  describe('On rendering', () => {
    it('should render properly', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
