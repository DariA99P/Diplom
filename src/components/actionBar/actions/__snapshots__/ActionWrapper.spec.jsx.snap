// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActionWrapper /> On rendering should render properly 1`] = `
<div>
  <Input
    disabled={false}
    prefixCls="ant-input"
    type="text"
  />
</div>
`;
